### ISSUE TITLE
(Login flow) when the user clears cookies/changes browser and logs in with his wallet for the second time, he should not see the invite code again

### ISSUE THREAD

**Issue by ddamazz** [TowneSquare/cred#61]
Current behavior:

1. User connects his wallet 0x123 for **the first time**
2. User inserts an invite code
3. User sees Cred points page
4. User disconnects his 0x123 wallet
5. User deletes cookies
6. Users wants to connect his 0x123 wallet for **the second time** and clicks on "Connect wallet"
_6. User has to insert invite code_
7. User sees Cred points page

Desired behavior:

1. User connects his wallet 0x123 for **the first time**
2. User inserts an invite code
3. User sees Cred points page
4. User disconnects his 0x123 wallet
5. User deletes cookies
6. Users wants to connect his 0x123 wallet for **the second time** and clicks on "Connect wallet"
7. User sees Cred points page

In short, when the user logs in with his wallet for the second time, he should not see the invite code again

**Comment by markots** [comment]
Done
